[2024-03-27T18:01:00.561+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T18:01:00.580+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_daily_table_test44.delete_daily_bigquery_external_table_task scheduled__2024-03-27T09:00:00+00:00 [queued]>
[2024-03-27T18:01:00.597+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_daily_table_test44.delete_daily_bigquery_external_table_task scheduled__2024-03-27T09:00:00+00:00 [queued]>
[2024-03-27T18:01:00.598+0800] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-03-27T18:01:00.621+0800] {taskinstance.py:1327} INFO - Executing <Task(BigQueryDeleteTableOperator): delete_daily_bigquery_external_table_task> on 2024-03-27 09:00:00+00:00
[2024-03-27T18:01:00.631+0800] {standard_task_runner.py:57} INFO - Started process 2364 to run task
[2024-03-27T18:01:00.636+0800] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'update_daily_table_test44', 'delete_daily_bigquery_external_table_task', 'scheduled__2024-03-27T09:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/flights_ingestion_dag.py', '--cfg-path', '/tmp/tmpryb4lcdj']
[2024-03-27T18:01:00.641+0800] {standard_task_runner.py:85} INFO - Job 7: Subtask delete_daily_bigquery_external_table_task
[2024-03-27T18:01:00.671+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T18:01:00.739+0800] {task_command.py:410} INFO - Running <TaskInstance: update_daily_table_test44.delete_daily_bigquery_external_table_task scheduled__2024-03-27T09:00:00+00:00 [running]> on host d10642315e48
[2024-03-27T18:01:00.759+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T18:01:00.844+0800] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_daily_table_test44' AIRFLOW_CTX_TASK_ID='delete_daily_bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T09:00:00+00:00'
[2024-03-27T18:01:00.847+0800] {bigquery.py:2348} INFO - Deleting: ny-rides-413103.flights_stg.external_table_daily_flights
[2024-03-27T18:01:00.849+0800] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-27T18:01:01.082+0800] {bigquery.py:1231} INFO - Deleted table ny-rides-413103.flights_stg.external_table_daily_flights
[2024-03-27T18:01:01.097+0800] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=update_daily_table_test44, task_id=delete_daily_bigquery_external_table_task, execution_date=20240327T090000, start_date=20240327T100100, end_date=20240327T100101
[2024-03-27T18:01:01.131+0800] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-03-27T18:01:01.166+0800] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-27T18:24:10.853+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T18:24:10.878+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_daily_table_test44.delete_daily_bigquery_external_table_task scheduled__2024-03-27T09:00:00+00:00 [queued]>
[2024-03-27T18:24:10.896+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_daily_table_test44.delete_daily_bigquery_external_table_task scheduled__2024-03-27T09:00:00+00:00 [queued]>
[2024-03-27T18:24:10.897+0800] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-03-27T18:24:10.930+0800] {taskinstance.py:1327} INFO - Executing <Task(BigQueryDeleteTableOperator): delete_daily_bigquery_external_table_task> on 2024-03-27 09:00:00+00:00
[2024-03-27T18:24:10.942+0800] {standard_task_runner.py:57} INFO - Started process 4830 to run task
[2024-03-27T18:24:10.954+0800] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'update_daily_table_test44', 'delete_daily_bigquery_external_table_task', 'scheduled__2024-03-27T09:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/flights_ingestion_dag.py', '--cfg-path', '/tmp/tmpy6whqjmn']
[2024-03-27T18:24:10.960+0800] {standard_task_runner.py:85} INFO - Job 16: Subtask delete_daily_bigquery_external_table_task
[2024-03-27T18:24:10.990+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T18:24:11.057+0800] {task_command.py:410} INFO - Running <TaskInstance: update_daily_table_test44.delete_daily_bigquery_external_table_task scheduled__2024-03-27T09:00:00+00:00 [running]> on host 51fbd2853be7
[2024-03-27T18:24:11.077+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T18:24:11.232+0800] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_daily_table_test44' AIRFLOW_CTX_TASK_ID='delete_daily_bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T09:00:00+00:00'
[2024-03-27T18:24:11.235+0800] {bigquery.py:2348} INFO - Deleting: flight-analytics-418504.flights_stg.external_table_daily_flights
[2024-03-27T18:24:11.237+0800] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-27T18:24:11.514+0800] {bigquery.py:1231} INFO - Deleted table flight-analytics-418504.flights_stg.external_table_daily_flights
[2024-03-27T18:24:11.532+0800] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=update_daily_table_test44, task_id=delete_daily_bigquery_external_table_task, execution_date=20240327T090000, start_date=20240327T102410, end_date=20240327T102411
[2024-03-27T18:24:11.602+0800] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-03-27T18:24:11.649+0800] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
